Tests/Realm.Tests/TransformHelpers.cs

```diff
@@ -0,0 +1,64 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Xml;
+using System.Xml.XPath;
+using System.Xml.Xsl;
+
+namespace Realms.Tests
+{
+    public static class TransformHelpers
+    {
+        [SuppressMessage("Security", "CA3075:Insecure DTD processing in XML", Justification = "The xml is static and trusted.")]
+        [SuppressMessage("Security", "CA5372:Use XmlReader For XPathDocument", Justification = "The xml is static and trusted.")]
+        public static void TransformTestResults(string resultPath, string transformFilePath)
+        {
+            var xpathDocument = new XPathDocument(resultPath);
+            var transform = new XslCompiledTransform();
+            transform.Load(transformFilePath);
+            using var writer = new XmlTextWriter(resultPath, null);
+            transform.Transform(xpathDocument, null, writer);
+        }
+
+        public static string CopyBundledFileToDocuments(string realmName, string destPath)
```

Comment:
Also, let's rename it to `ExtractBundledFile` and change the first parameter to be `fileName` rather than `realmName` since it's not used only for realm files anymore.